reference/camera.md

```diff
@@ -106,6 +107,12 @@ no focus is available on the camera device.
 with a controllable zoom system. If this field is set to NULL, then no zoom is
 available on the camera device.
 
+- The `recognition` field may contain a [Recognition](recognition.md) node to provide the camera device with object recognition capability.
+The camera can only recognize [Solid](solid.md) nodes whose `recognitionColors` is not empty.
+The object size is estimated using the `boundingObject` of the [Solid](solid.md) and the ones of all its children.
+Note that the returned size is an estimation and can be in some cases overestimated.
+In case the [Solid](solid.md) and its children doesn't have any bounding object, the dimension is estimated using the shape, this estimation is usually widely overestimated.
```

Comment:
"and its children" needs parentheses if you wish to use doesn't instead of don't here:

In case the [Solid](solid.md) (and its children) doesn't have..

OR

In case the [Solid](solid.md) and its children don't have..